src/desktop/apps/order2/routes.tsx

```diff
@@ -4,7 +4,7 @@ import { buildServerApp } from "reaction/Artsy/Router/server"
 import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
 import { routes } from "reaction/Apps/Order/routes"
 import { stitch } from "@artsy/stitch"
-import metaphysics from "lib/metaphysics.coffee"
+const metaphysics = require("lib/metaphysics.coffee")
```

Comment:
oh, this is interesting... what's happening here?

Reply:
Typescript doesn't recognize the `.coffee` extension, but we've been conventionally getting around it with a normal require (not sure if there's a better way).

Reply:
Maybe we can define an ambient declaration for `*.coffee` and have it export `any` like the advice was for JSON files

Reply:
I'd normally do a `ts-ignore`, but yeah, this works.